packages/card/style.ts

```diff
@@ -19,6 +20,16 @@ export const cardBase = css`
   }
 `;
 
+export const cardHeaderImage = css`
+  > img {
+    width: 100%;
+    height: 30%;
+    max-height: 100px;
```

Comment:
❓ - Would we want this to be more flexible in the future?

Reply:
This I was debating over as well. Do you think a user would prefer to be able to set the height of the image or have it taken care of for them?